opm/simulators/flow/KeywordValidation.cpp

```diff
@@ -33,9 +33,26 @@
 namespace Opm
 {
 
+
+
 namespace KeywordValidation
 {
 
+
```

Comment:
It would probably be better to have special validation files in a separate file, to separate it from the generic functionality of the validation class. But that can be done when the number of these functions grows.

Reply:
I'll Split as suggested